<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminMenu</class>
 <widget class="QWidget" name="AdminMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>392</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>770</width>
    <height>500</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(85, 170, 127);

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">QFrame {
    background-color: #ccffcc;
    border-radius: 15px;
    padding: 3px;
}
QPushButton {
    background-color: #55aa7f;
    border-radius: 10px;
    padding: 10px;
    font-size: 20px;
    color: #b1ff00;
                }
QPushButton:hover {
    background-color: #99ff99;
    color: #55aa7f;
 }
QPushButton:pressed {
    background-color: #66ff66;
}
            

                </string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="maximumSize">
           <size>
            <width>110</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>../resorces/logo.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">color: #006400;              /* √áok koyu ye≈üil: DarkGreen */
font: bold 16pt &quot;Segoe UI&quot;;  /* Kalƒ±n ve 16 punto Segoe UI */

</string>
          </property>
          <property name="text">
           <string>Preferences Menu - Admin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_exit">
          <property name="maximumSize">
           <size>
            <width>110</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 15px;
padding: 0px;</string>
          </property>
          <property name="text">
           <string>‚ùå Exit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item row="1" column="1">
         <widget class="QPushButton" name="btn_admin_menu">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 30px;</string>
          </property>
          <property name="text">
           <string>üõ†Ô∏è Admin Menu</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btn_mentor">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 30px;</string>
          </property>
          <property name="text">
           <string>üë©‚Äçüè´ Mentor Meeting</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="btn_applications">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 30px;</string>
          </property>
          <property name="text">
           <string>üì¶ Applications</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="btn_interviews">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 30px;</string>
          </property>
          <property name="text">
           <string>üó£Ô∏è Interviews</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
